src/styles/zIndex.js

```diff
@@ -12,4 +12,5 @@ export default {
   popover: 2100,
   snackbar: 2900,
   tooltip: 3000,
+  mobileStepper: 1100,
```

Comment:
I think you are referring to [this conversation](https://github.com/callemall/material-ui/issues/7033#issuecomment-305796977) but this is not related - I've done this because I envisaged this being sticky & fixed to the bottom of the mobile device.